cg/store/filters/status_case_sample_filters.py

```diff
@@ -56,6 +57,25 @@ def get_not_sequenced_cases(case_samples: Query, **kwargs) -> Query:
     return case_samples.filter(Sample.last_sequenced_at == None).distinct()
 
 
+def get_sequenced_cases(case_samples: Query, **kwargs) -> Query:
```

Comment:
Let's add some unit tests to these too

Reply:
We will end up refactoring the order summary logic extensively and these queries will not live past that, so I'll skip writing tests for these queries.